### PR TITLE
chore: Splitbutton WinUI alignment

### PR DESCRIPTION
**Pull request by ramezgerges**
GitHub Issue (If applicable): closes #

## PR Type

What kind of change does this PR introduce?
- Refactoring (no functional changes, no api changes)

### <samp>🤖[[deprecated]](https://githubnext.com/copilot-for-prs-sunset) Generated by Copilot at 15f6aa5</samp>

This pull request adds and updates several files related to the `SplitButton` and `ToggleSplitButton` controls, such as test pages, automation peers, and visual state viewers. The main purpose of this pull request is to improve the compatibility, consistency, and testability of these controls with the WinUI source code.

## PR Checklist

Please check if your PR fulfills the following requirements:

- [ ] Docs have been added/updated which fit [documentation template](https://github.com/unoplatform/uno/blob/master/doc/.feature-template.md) (for bug fixes / features)
- [ ] [Unit Tests and/or UI Tests](https://github.com/unoplatform/uno/blob/master/doc/articles/uno-development/working-with-the-samples-apps.md) for the changes have been added (for bug fixes / features) (if applicable)
- [ ] Validated PR `Screenshots Compare Test Run` results.
- [ ] Contains **NO** breaking changes
- [ ] Associated with an issue (GitHub or internal) and uses the [automatic close keywords](https://help.github.com/en/github/managing-your-work-on-github/linking-a-pull-request-to-an-issue).
- [ ] Commits must be following the [Conventional Commits](https://www.conventionalcommits.org/en/v1.0.0/#summary) specification.

## Other information

Internal Issue (If applicable):

### CODE REVIEW

**Comment by ramezgerges** [comment]
I keep getting `The type local:SplitButton could not be found` after rebasing on top of master (after the namespace changes). `themeresources_v2.xaml` contains this (after the generation):
```xaml
<Style TargetType="local:SplitButton" BasedOn="{StaticResource SplitButtonStyle}" />
<Style TargetType="local:ToggleSplitButton" BasedOn="{StaticResource SplitButtonStyle}" />
<Style TargetType="controls:SplitButton" BasedOn="{StaticResource SplitButtonStyle}" />
<Style TargetType="controls:ToggleSplitButton" BasedOn="{StaticResource SplitButtonStyle}" />
```

The first 2 (wrong) lines are pre-existing. The last 2 are newly generated.

@Youssef1313

File: src/Uno.UI/UI/Xaml/Controls/Popup/Popup.cs [hunk 1]
```diff
@@ -311,4 +311,7 @@ internal Brush LightDismissOverlayBackground
 
 	internal static DependencyProperty LightDismissOverlayBackgroundProperty { get; } =
 		DependencyProperty.Register(nameof(LightDismissOverlayBackground), typeof(Brush), typeof(Popup), new FrameworkPropertyMetadata(defaultValue: null, propertyChangedCallback: (o, e) => ((Popup)o).ApplyLightDismissOverlayMode()));
+
+	// On WinUi, a popup is not IsTabStop but is somehow focusable. This is a workaround to match that behaviour.
+	internal override bool IsFocusableForFocusEngagement() => true;
```
Comment (ramezgerges) [inline src/Uno.UI/UI/Xaml/Controls/Popup/Popup.cs:316]: @MartinZikmund I'm completely confused at this point. In #13734, we confirmed that popups are indeed not IsTabStop. However, it seems that when nothing is focusable inside the popup, the popup ends up being focused anyway? This is necessary to make popups closable on Escape (which is blocking this PR). Note that this is the case in general, not only for popups or flyouts. Check out the  Popup_Simple sample and try to close the popups using Escape. It won't work. With this workaround, it does.

**Comment by jeromelaban** [comment]
@ramezgerges could you adjust for the conflict?

**Review by jeromelaban — APPROVED** [review]

**Comment by ramezgerges** [comment]
@jeromelaban Rerunning CI keeps failing on `When_MediaPlayerElement_SetSource_Check_Play` which have been disabled in #15472 by now. Should we just merge?

**Pull request merged**